test/EndToEnd/tests/install.ps1

```diff
@@ -56,6 +56,23 @@ function Test-InstallPackageWithInvalidHttpSource {
 	Assert-Throws { Install-Package $package -ProjectName $project.Name -source $source } $message
 }
 
+function Test-InstallPackageWithInvalidHttpSourceVerbose {
+	# Arrange
+	$package = "Rules"
+	$project = New-ConsoleApplication
+	$source = "http://example.com"
+	$url = $source+"/FindPackagesById()?id='$package'"
+	$errorcode = "404 Not Found"
+	$message = "  GET $url   NotFound $url ? An error occurred while retrieving package metadata for '$package' from source '$source'."+
+               "  The V2 feed at '$url' returned an unexpected status code '$errorcode'. Unable to find package 'Rules' at source '$source'."
```

Comment:
Actually it'd be better to escape the whole thing altogether with `[regex]::Escape($message)`